test/Interactive/fixture/issue3029/SConstruct

```diff
@@ -0,0 +1,23 @@
+import os, sys
+
+DefaultEnvironment(tools=[])
+env = Environment()
+
```

Comment:
Can you add a see README.md here?

Reply:
For ?

Reply:
The code below which is explained in the README.md and doesn't make much sense without an explanation?

Reply:
Oh, I see, you want a pointer to that file. But then that should be added to every test which does this, right? Seems a bit of a pain (20-something of them). The README is there if someone needs to dig.

Reply:
I PR'd your branch with such change. Please merge unless you really oppose it.